cfgov/jinja2/v1/_includes/molecules/_vars-content.html

```diff
@@ -0,0 +1,9 @@
+{% macro _privacy_statement() %}
+<p class="short-desc">
```

Comment:
Whichever way this goes, maybe add the consensus to https://github.com/cfpb/front-end/issues/75